data-showcase/src/main/user-interface/src/app/shopping-cart/shopping-cart.component.ts

```diff
@@ -35,6 +36,7 @@ export class ShoppingCartComponent implements OnInit {
           return item['name'];
         });
       });
+    this.dialogWidth = window.screen.width - 200
```

Comment:
It would be nice if we can also update the width when the window resizes, but the dialog component of PrimeNG does not seem to be suitable for that, alas.